requirements.txt

```diff
@@ -1,60 +1,2 @@
-appdirs==1.4.3
-appnope==0.1.0
-astroid==2.0.4
-attrs==18.2.0
-backcall==0.1.0
-black==18.6b4
-certifi==2018.8.24
-chardet==3.0.4
-chromedriver-binary==2.41.0
-click==6.7
-dash==0.32.0
-dash-core-components==0.40.2
-dash-html-components==0.13.2
-dash_renderer==0.15.1
-decorator==4.3.0
-flake8==3.5.0
-Flask==1.0.2
-Flask-Compress==1.4.0
-gunicorn==19.9.0
-idna==2.7
-ipdb==0.11
-ipython==6.5.0
-ipython-genutils==0.2.0
-isort==4.3.4
-itsdangerous==0.24
-jedi==0.12.1
-Jinja2==2.10
-jsonschema==2.6.0
-jupyter-core==4.4.0
-lazy-object-proxy==1.3.1
-MarkupSafe==1.0
-mccabe==0.6.1
-nbformat==4.4.0
-numpy==1.15.1
-pandas==0.23.4
-parso==0.3.1
-percy==2.0.0
-pexpect==4.6.0
-pickleshare==0.7.4
-plotly==3.2.1
-prompt-toolkit==1.0.15
-ptyprocess==0.6.0
-pycodestyle==2.3.1
-pyflakes==1.6.0
-Pygments==2.2.0
-pylint==2.1.1
-python-dateutil==2.7.3
-pytz==2018.5
-requests==2.19.1
-retrying==1.3.3
-selenium==3.14.0
-simplegeneric==0.8.1
-six==1.11.0
-toml==0.9.6
-traitlets==4.3.2
-typed-ast==1.1.0
-urllib3==1.23
-wcwidth==0.1.7
-Werkzeug==0.14.1
-wrapt==1.10.11
+-r requirements-base.txt
+-r requirements-v1.txt
```

Comment:
@alexcjohnson So interestingly Heroku does not seem super flexible about its setup procedure. The project has been tagged as Python and as far as I understand, that means it will look for setup.py, runtime.txt and requirements.txt -- so whatever weird multi version setup we have, we must play around that limitation.

Reply:
Annoying, but nice solution!